src/main/java/Duke.java

```diff
@@ -1,10 +1,240 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Duke.
+ * @author Thomas Hogben
+ */
 public class Duke {
+    private static final String MSG_ERROR_DEFAULT = "im sorry I is no understand.";
+    private static final String MSG_ERROR_BLANK_DESCRIPTION = "is no leave description blank;";
+    private static final String MSG_ERROR_INVALID_TASK_NUMBER = "what kind of number is (||❛︵❛.)";
+    private static final String MSG_ERROR_NOT_ENOUGH_TASKS = "we is dont have that many tasks yet.";
+    private static final String MSG_ERROR_UNSPECIFIED_TASK = "please is specify task please,";
+    private static final String MSG_EXIT = "okay is bye!!";
+    private static final String MSG_NO_TASKS = "is no tasks today.";
+    private static final String MSG_TASK_ADDED = "is added.";
+    private static final String MSG_TASK_DELETED = "is deleted!";
+    private static final String MSG_TASK_DONE = "is done!";
+
+    /** Program loops until this is set to false */
+    private static boolean running = true;
```

Comment:
isRunning would be a more appropriate name to follow the coding standard in my opinion